src/Symfony/Component/DependencyInjection/Tests/Fixtures/xml/services31.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="foo" class="Foo">
+            <getter name="getBar" type="collection">
+                <getter key="bar" type="service" id="bar" />
```

Comment:
Just saw it was the same for `<property>`... but still feel weird to read for getter injection to me. 😅